pkg/ddc/alluxio/transform.go

```diff
@@ -242,12 +242,6 @@ func (e *AlluxioEngine) transformCommonPart(runtime *datav1alpha1.AlluxioRuntime
 
 	e.transformShortCircuit(runtimeInfo, value)
 
-	clusterDomain, err := common.GetClusterDomain()
-	if err != nil {
-		return err
-	}
-	value.ClusterDomain = clusterDomain
```

Comment:
Do you want to still keep attribute `ClusterDomain`?

Reply:
If this variable is not used, I recommend deleting it

Reply:
So please delete ClusterDomain in this PR. Thanks.